synth/src/cli/import_utils.rs

```diff
@@ -88,7 +85,10 @@ fn populate_namespace_primary_keys<T: DataSource + RelationalDataSource>(
                 table_name, primary_key.column_name
             ))?;
             let node = namespace.get_s_node_mut(&field)?;
-            *node = Content::Number(NumberContent::U64(U64::Id(Id::default())));
+            *node = Content::Unique(UniqueContent {
```

Comment:
One problem comes as a result of this and the fact that, for postgres at least, integers are mapped to ranges.

Because these ranges grow through merging row values, import usually gives you small ranges. This implies that you can't generate more than ~10 rows using the schema generated by import on integer primary keys, even when using the `--size` argument.

Why don't we have type-dependent match here? For `int` primary keys, go with `id` or default integer range (which spans the whole of integers). For `varchar` go with the pattern, etc.

Reply:
Yeah I was thinking about this.

In the description I've put as 'future work' for us to support the `Serial` type which *should* get mapped to `id`.

Furthermore, it is not necessarily true that integer primary keys are auto incrementing. But I do see the point about us exhausting the range.

> Why don't we have type-dependent match here?

```rust
match node {
   Content::Number(n) => // map to int::id,
   Content:: String(s) => // map to string::pattern
   etc...
}
```

Is this what you mean?